examples/bpk-component-chip-group/stories.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import BpkChipGroup, {
+  BpkChipGroupSingleSelect,
+  BpkChipGroupState,
+  BpkChipGroupSingleSelectState,
+} from '../../packages/bpk-component-chip-group';
+
+import {
+  BpkChipGroupRail,
+  BpkChipGroupWrapping,
+  BpkChipGroupSticky,
+  BpkSingleChipGroupWrapping,
+  OnDarkChipGroup,
+  OnImageChipGroup,
+  BpkChipGroupWithLabel,
+  MixedExample,
+  AllChipTypesGroup,
+  OnContrastChipGroup,
+  StateManagement,
+} from './examples';
+
+export default {
+  title: 'bpk-component-chip-group',
+  component: BpkChipGroup,
+  subcomponents: {
+    BpkChipGroupSingleSelect,
+    BpkChipGroupState,
+    BpkChipGroupSingleSelectState,
+    // TODO: can we show the shape of ChipItem here?
```

Comment:
Is this still required?

Reply:
Ah gotcha! This should be possible to do, as we have the documentation part which we can add subcomponents to! Let me check if for types it is possible but should be as we have it for other components!